trajan/readers/omb.py

```diff
@@ -245,7 +249,6 @@ def read_omb_csv(path_in: Path,
                          attrs={
                              "_FillValue": "NaN",
                              "standard_name": "latitude",
-                             "units": "degree_north",
```

Comment:
Is there a reason why units was removed? It seemed correct?

Reply:
ah, I was not aware that there is a default filling of e.g. units. Perhaps added by Gaute.
But this is eventually good, and yes, we should then avoid adding it twice.

Reply:
Ok, not to my understanding, but I might also be wrong.
But I believe it is always good to add units in any case. Even if some softwares might assume a "default unit", other softwares (or users/persons) might not, and could give rise to doubt/uncertainty.

Reply:
I tried to find out, but it seems that there is no default value of `units`, thus it should always be added:
https://github.com/cf-convention/cf-conventions/issues/166
It should be possible to convert the given unit to the "Canonical unit"; e.g. if "Canonical unit" is "Kelvin", it is ok to have units = "Celsius", as "Celcius" can be converted to "Kelvin"